README.md

```diff
@@ -1,2 +1,81 @@
 # rokt-demo-ios
-Rokt iOS Demo Application
+Rokt Demo application is a sample app built to showcase Rokt SDK functionality.
+
+## Requirements
+
+XCode 13 and above. Project is configured to run on iOS 13.0 and above.
+
+## Project architecture
+
+This project it is implemented based on MVVM pattern with SwiftUI. 
```

Comment:
remove "it". or This project is implemented with MVVM pattern using SwiftUI.

Reply:
Done